docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: ../app/.
```

Comment:
I think it would be simpler if we use git submodules instead of relying on the users file structure

Reply:
I also think this can be replaced by a more strict way to setup the repo. The infra solution was a fast solution that already involved multiple repos. I'm willing to change this part but we should make a sync before I go any further, what do you think?